<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;DejaVu Sans&quot;;color:rgb(174,0,0)</string>
   </property>
   <property name="text">
    <string>Create A New Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>581</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;DejaVu Sans&quot;; color:rgb(174,0,0)</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ph_create">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>610</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_email">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 236);
font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_passwd">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 236);
font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>301</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_firstname">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 236);
font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_lastname">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 236);
font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>261</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPage">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>710</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>680</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Do you have an account?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ph_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
